### DISCUSSION TITLE
2022-05: 平静的春节

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#329] [Announcements]
# 2022-05: 平静的春节

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-05/

**Comment by hualet** [comment]
这个游戏不违法么…？

**Reply by Xuanwo** [reply]
不太懂法律，简单搜索了一下相关的条文，大概是适用这一条：

> 不以营利为目的，进行带有少量财物输赢的娱乐活动，以及提供棋牌室等娱乐场所只收取正常的场所和服务费用的经营行为等，不以赌博论处。

来源: [最高人民法院最高人民检察院关于办理赌博刑事案件具体应用法律若干问题的解释](https://www.waizi.org.cn/doc/24480.html)

**Reply by Xuanwo** [reply]
不过关于 JJ 斗地主涉赌的新闻也是有的：[“JJ网游”被指涉赌：一玩家输掉50万后妻离子散](http://www.xinhuanet.com/politics/2015-06/23/c_127939036.htm)

**Reply by Xuanwo** [reply]
大家打打免费的场次就得了，可别上头赌大钱呀

**Comment by GrapeBaBa** [comment]
我今年也在玩JJ呢
